backend/core/domains/payments/services/invoice_service.py

```
@@ -201,15 +212,476 @@ def create_from_quote(quote, due_days=14):
+                    ).first()
+
+                    if existing_transaction:
+                        # Reuse existing payment intent
+                        logger.info(f"Reusing existing transaction {existing_transaction.transaction_id} for payment {payment.payment_number}")
+
+                        try:
+                            # Retrieve existing Stripe payment intent
+                            intent = stripe.PaymentIntent.retrieve(existing_transaction.transaction_id)
+                            transaction_record = existing_transaction
+                        except stripe.error.StripeError as e:
+                            logger.error(f"Failed to retrieve existing Stripe payment intent: {e}")
```

Ruff (0.13.3)

Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)